src/components/Steps/studentDetails.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+
+export default function StudentDetails({ setActiveStep }) {
+  return (
+    <>
+      <div className="bg-white rounded-2 border border-solid border-[#B52326] m-10 rounded-lg">
+        <form action="" className="flex flex-col items-center m-[60px]">
+          <select className="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded block w-full p-2.5">
+            <option selected>Program</option>
+            <option value="Haryana Students">Haryana Students</option>
```

Comment:
this isn't the right way to structure these things. 

a better way to structure is to have a separate file / `constants.js` file for all possible options for this dropdown. and here you can have a for loop or something like that. see how category dropdown is provided here https://github.com/avantifellows/college-predictor/blob/main/constants.js